app/src/main/java/com/hedvig/app/ui/compose/composables/ErrorDialog.kt

```diff
@@ -0,0 +1,33 @@
+package com.hedvig.app.ui.compose.composables
+
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.res.stringResource
+import com.hedvig.app.R
+
+@Composable
+fun ErrorDialog(
+    show: MutableState<Boolean>,
```

Comment:
Since we only need the "setting" part of the mutableState and we never use `show.value` here, wouldn't this be better off as a callback to change the state that is hoisted up?

Reply:
Good idea